core/chains/evm/gas/block_history_estimator.go

```diff
@@ -96,12 +111,16 @@ func NewBlockHistoryEstimator(lggr logger.Logger, ethClient evmclient.Client, cf
 		chainID,
 		cfg,
 		make([]Block, 0),
+		sync.RWMutex{},
```

Comment:
nit/ and sort of out of scope: this seems like a strong candidate for named fields instead of passing all these zero value (especially the mutexes). It looks like 8 or 9 lines could be removed.

Reply:
I quite like non-named fields in constructors because it ensures you specified all of them